### ISSUE TITLE
Convert from GNU Make to Snakemake/Conda for building site

### ISSUE THREAD

**Issue by pschloss** [UMCarpentries/intro-curriculum-r#177]
I've been banging my head against the desk trying to get the site to build on my computer. There's issues with the version of ruby, gems, etc. I wonder if things wouldn't be a lot easier in the long run to build this as a snakemake workflow from within a conda environment. I also feel like there's a lot of overhead within that `Makefile` that we probably don't need.

Any thoughts?

**Comment by kelly-sovacool** [comment]
Agreed, there is a TON of overhead with the `Makefile` and the general approach to building the site. I can usually build individual lessons, but getting the whole site to build is a bear.

I suspect that The Carpentries' will have fixed this issue in the new lesson infrastructure that's currently being rolled out (#174). I'm not sure how labor intensive it will be to switch to the new style.
